platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/reconnect/TeacherPullVirtualTreeView.java

```diff
@@ -85,8 +92,15 @@ public TeacherPullVirtualTreeView(
         this.reconnectConfig = reconnectConfig;
         new ThreadConfiguration(threadManager)
                 .setRunnable(() -> {
-                    records = pipeline.pausePipelineAndRun("copy", root::detach);
-                    ready.countDown();
+                    try {
+                        records = pipeline.pausePipelineAndRun("copy", root::detach);
+                    } catch (final Throwable t) {
```

Comment:
Catching and rethrowing an exception seems redundant here. The exception was properly caught and logged (3 times actually) in the calling context.